### PR TITLE
main: print tx stats log

### PR DESCRIPTION
**Pull request by thibault-martinez**
The main is now printing tx stats logs.

Sample: `2018-11-05 12:04:21:           main:   INFO: Transactions: to process 0, to broadcast 0, to request 49, count 3956`

Depends on #470, will rebase after merge.

Should make #260 easier to implement.

# Test Plan:
Not applicable.

### CODE REVIEW

File: ciri/main.c
Comment (th0br0) [inline ciri/main.c:75]: This breaks shutdown because it won't ever be interrupted?

**Review by th0br0 — CHANGES_REQUESTED** [review]

File: ciri/main.c
Comment (thibault-martinez) [reply ciri/main.c:75]: It temporarily replaces a `sleep` with an arbitrary value so until #260 is merged we won't really have a sane shutdown.

Comment (thibault-martinez) [reply ciri/main.c:75]: We can wait until #260 is merged to merge this one.

**Review by th0br0 — APPROVED** [review]

**Pull request merged**
